k8s/prod/common/bsp/bulk-scan-processor.yaml

```diff
@@ -28,8 +28,9 @@ spec:
         STORAGE_URL: https://bulkscan.platform.hmcts.net
         PROCESS_PAYMENTS_ENABLED: true
         SMTP_HOST: smtp.office365.com
-        SCAN_DELAY: 300000
-        REUPLOAD_DELAY: 600000
+        SCAN_DELAY: "300000"
```

Comment:
no quotes needed :)

Reply:
figured juuuuust in case it'll start to convert to scientific numbers like we had on another config :D